ui/raidboss/data/07-dt/dungeon/origenics.txt

```diff
@@ -0,0 +1,220 @@
+### ORIGENICS
+# ZoneId: 1208
+
+hideall "--Reset--"
+hideall "--sync--"
+
+# .*is no longer sealed
+0.0 "--Reset--" SystemLogMessage { id: "7DE" } window 0,100000 jump 0
+
+#~~~~~~~~~~~~#
+# Herpekaris #
+#~~~~~~~~~~~~#
+
+# -ii 8E71 8E72 8E73 8E7A
+# -p 8EA7:1011.1
+
+# The Resource Transport Elevator will be sealed off
+1000.0 "--sync--" SystemLogMessage { id: "7DC", param1: "139C" } window 1000,1
+1006.1 "--sync--" StartsUsing { id: "8EA7", source: "Herpekaris" }
+1011.1 "Strident Shriek" Ability { id: "8EA7", source: "Herpekaris" }
+1017.3 "--north--" Ability { id: "8EA8", source: "Herpekaris" }
+1021.7 "Vasoconstrictor" Ability { id: "8E6B", source: "Herpekaris" }
+1022.8 "Poison Heart (puddle drop)" Ability { id: "8E6C", source: "Herpekaris" }
+# combine Venomspill + Pod Burst, which happen at the same time
+1029.8 "Venomspill + Pod Burst 1" Ability { id: "9676", source: "Herpekaris" }
+1034.9 "Venomspill + Pod Burst 2" Ability { id: "9677", source: "Herpekaris" }
+1039.9 "Venomspill + Pod Burst 3" Ability { id: "9677", source: "Herpekaris" }
+1045.0 "Venomspill + Pod Burst 4" Ability { id: "9677", source: "Herpekaris" }
+
+# begin loop
+1049.2 "--center--" Ability { id: "8EA8", source: "Herpekaris" }
+1059.5 "Writhing Riot" Ability { id: "8E6F", source: "Herpekaris" }
+1059.8 "Right Sweep/Left Sweep/Rear Sweep 1" #Ability { id: ["8E75", "8E76", "8E77"], source: "Herpekaris" } window 1.5,1.5
+1061.8 "Right Sweep/Left Sweep/Rear Sweep 2" #Ability { id: ["8E75", "8E76", "8E77"], source: "Herpekaris" } window 1.5,1.5
+1063.8 "Right Sweep/Left Sweep/Rear Sweep 3" #Ability { id: ["8E75", "8E76", "8E77"], source: "Herpekaris" } window 1.5,1.5
```

Comment:
This might be worth combining into an EN replaceSync to be just `Sweep X`.

Reply:
Right now, the replaceSync makes it `Right/Left/Rear Sweep X` -- i thought having at least some directional reference in there would be useful, but I can remove the `Left/Right/Rear` portion if you think `Sweep` is good enough.

Reply:
No, you know what it is, I completely missed that you had already done it. This is good as it is then.